demo/title/multisize/playground

```diff
@@ -0,0 +1,13 @@
+return (
+    <div>
+        <TitleMultisize preTitle="desde" title="250.000 €"  postTitle={['1.234 €/m', <sup>2</sup>]} titleSize="l"/>
```

Comment:
Executing this, you got that ugly `Each child in an array or iterator should have a unique "key" prop. Check the render method of TitleMultisize. See https://fb.me/react-warning-keys for more information` warning, looks like adding key attribute to the `<sup>` tags fixes it